.github/scripts/bump.sh

```
@@ -0,0 +1,225 @@
+  exit 1
+fi
+
+# Check if uv is available
+if [ ! -x "$UV_BIN" ]; then
+  printf "%b[ERROR] uv not found at %s. Run 'make install-uv' first.%b\n" "$RED" "$UV_BIN" "$RESET"
+  exit 1
+fi
+
+# Helper function to prompt user to continue
+prompt_continue() {
+  local message="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)